arch/arm64/src/imx9/Kconfig

```diff
@@ -28,4 +28,8 @@ config IMX9_UART1
 	select UART1_SERIALDRIVER
 endmenu # iMX Peripheral Selection
 
+config IMX9_PLL
+	bool "PLL setup support (WIP)"
```

Comment:
maybe no need to say WIP here, it just needs to be removed later anyways....

Reply:
It's in upstream already, let's remove it when the PLL support is finalized